proto/storage/base_image.proto

```diff
@@ -8,9 +8,8 @@ option go_package = "./storage;storage";
 option java_package = "io.stackrox.proto.storage";
 
 message BaseImage {
-  string id = 1; // @gotags: search:"Base Image Id,store,hidden" sql:"pk"
-  // TODO ROX-32114 add base image repo table and add FK to base image repo table
-  string base_image_repository_id = 2; // @gotags: search:"Base Image Repository Id,store"
+  string id = 1; // @gotags: search:"Base Image Id,store,hidden" sql:"pk,type(uuid)"
+  string base_image_repository_id = 2; // @gotags: sql:"fk(BaseImageRepository:id)"
   string repository = 3; // @gotags: search:"Base Image Repository,store"
```

Comment:
Repository here helps with one less JOIN, or can we drop it?

Reply:
I would agree with the argument "it helps us to avoid one JOIN in some case"